test/Microsoft.AspNetCore.Authentication.Test/OpenIdConnect/MockOpenIdConnectMessage.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+
+namespace Microsoft.AspNetCore.Authentication.Tests.OpenIdConnect
+{
+    internal class MockOpenIdConnectMessage : OpenIdConnectMessage
```

Comment:
Many of the usages of Mock that used to be here were 10x more code than alternatives. Most of the data structures we use are mutable and don't require mocking. You could use Mock, but with reasonable restraint please.